CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2024-09-23
```

Comment:
Kan je het versienummer ophogen naar 2.0? Door het verwijderen van de obsolete functies is het een backwards compatibality break geworden en is een nieuwe major nodig (zie https://semver.org/).

Ik mis ook deze verwijzing in de changelog en die is niet onbelangrijk ;)